docs/rum_collection.md

```diff
@@ -265,6 +280,50 @@ Global.rum.addError(message: "error message.")
 
 For more details and available options, refer to the code documentation comments in `DDRUMMonitor` class.
 
+## Data scrubbing
+
+To modify the attributes of a RUM event before it is sent to Datadog or to drop an event entirely, use the event mappers API when configuring the SDK:
+```swift
+Datadog.Configuration
+    .builderUsing(...)
+    .setRUMViewEventMapper { viewEvent in 
+        return viewEvent
+    }
+    .setRUMErrorEventMapper { errorEvent in
+        return errorEvent
+    }
+    .setRUMResourceEventMapper { resourceEvent in
+        return resourceEvent
+    }
+    .setRUMActionEventMapper { actionEvent in
+        return actionEvent
+    }
+    .build()
+```
+Each mapper is a Swift closure with a signature of `(T) -> T?`, where `T` is a concrete RUM event type. This allows changing portions of the event before it gets sent. For example to redact sensitive information in RUM Resource's `url` you may implement a custom `redacted(_:) -> String` function and use it in `RUMResourceEventMapper`:
```

Comment:
After RUMM-1077 and #415, we need to update that part to note that `RUMViewEventMapper` must return a non-optional `T` and `RUMViewEvent` can't be dropped (they will all be sent to Datadog).

Reply:
Done 👍. I also included a note on the `view.name` mutability coming in #418.